ui/app/routes/loading.js

```diff
@@ -1,25 +1,9 @@
 import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
 
 export default Route.extend({
-  router: service(),
-  init() {
+  setupController(controller) {
     this._super(...arguments);
-    this.router.on('routeWillChange', (transition) => {
-      this.set('myTargetRouteName', transition.to.name);
-    });
```

Comment:
This pattern wasn't working when I tested prior to making the change since the router updates for intermediate transitions so `loading` was the value for `transition.to.name`. This resulted in the standard `LogoSplash` displaying rather than the OIDC callback. Using `location.pathname` solves the issue and the new `OidcCallbackSplash` component is displayed which prevents a change in display while transition from the loading state to the oidc-callback route.